students/carlos_novoa/lesson02/fizzbuzz.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+"""
+Lesson2, FizzBuz
+"""
+
+
+def fizzbuzz():
+    for i in range(1, 101):
+        if (i % 5 == 0 and i % 3 == 0):
+            print('FizzBuzz')
+        elif (i % 3 == 0):
+            print('Fizz')
+        elif (i % 5 == 0):
+            print('Buzz')
+        else:
+            print(i)
```

Comment:
FizzBuzz looks good. I don't your assignment submission on the course website for this one, though.